Integrations/SplunkPy/SplunkPy.yml

```
@@ -346,7 +351,7 @@ script:
       JSON event protocol.
     execution: false
     name: splunk-submit-event-hec
-  dockerimage: demisto/splunksdk:1.0
+  dockerimage: demisto/splunksdk:2.0
```

this will fix the build
```suggestion
  dockerimage: demisto/splunksdk:2.0
  dockerimage45: demisto/splunksdk:1.0
```